08-transport.md

```diff
@@ -5,7 +5,7 @@ provide confidentiality for all transcripts between nodes and is authenticated i
 avoid malicious interference. Each node has a known long-term identifier that
 is a public key on Bitcoin's `secp256k1` curve. This long-term public key is
 used within the protocol to establish an encrypted and authenticated connection
-with peers, and also to authenticate any information advertised on behalf
+with peers and also to authenticate any information advertised on behalf
```

Comment:
I think the (oxford) comma is warranted here since we have a list of lists, isn't it?

Reply:
+1

Reply:
Sounds good; reverted.